tests/build_/test_arrangements.py

```diff
@@ -1703,3 +1729,319 @@ def test_export_operator_manifests(self, osbs, extract_platform):
+        _, build_json = self.get_build_request(build_type, osbs, additional_params)
+        plugins = get_plugins_from_build_json(build_json)
+
+        args = plugin_value_get(plugins, 'postbuild_plugins', 'tag_from_config', 'args')
+
+        assert set(args.keys()) == set(['tag_suffixes'])
+        assert set(args['tag_suffixes'].keys()) == set(['unique', 'primary', 'floating'])
+
+        unique_tags = args['tag_suffixes']['unique']
+        assert len(unique_tags) == 1
+        unique_tag_suffix = ''
+        if has_plat_tag:
+            unique_tag_suffix = '-' + additional_params.get('platform')
+        assert unique_tags[0].endswith(unique_tag_suffix)
+
+        primary_tags = args['tag_suffixes']['primary']
+        if has_primary_tag:
+            assert set(primary_tags) == set(['{version}-{release}'])
+
+    @pytest.mark.parametrize('osbs', [  # noqa:F811
+        {
+            'additional_config': get_pulp_additional_config(),
+            'kwargs': {'registry_uri': 'registry.example.com/v2'},
+        }
+    ], indirect=True)
+    def test_pulp_push(self, osbs):
+        additional_params = {
+            'base_image': 'fedora:latest',
+        }
+        _, build_json = self.get_worker_build_request(osbs, additional_params)
+        plugins = get_plugins_from_build_json(build_json)
+
+        args = plugin_value_get(plugins, 'postbuild_plugins', PLUGIN_PULP_PUSH_KEY, 'args')
+
+        expected_args = {
+            'load_exported_image': True,
+            'publish': False,
+        }
+
+        assert args == expected_args
+
+    @pytest.mark.parametrize('osbs', [OSBS_WITH_PULP_PARAMS], indirect=True)  # noqa:F811
+    def test_pulp_tag(self, osbs):
+        additional_params = {
+            'base_image': 'fedora:latest',
+        }
+        _, build_json = self.get_orchestrator_build_request(osbs, additional_params)
+        plugins = get_plugins_from_build_json(build_json)
+
+        get_plugin(plugins, 'postbuild_plugins', PLUGIN_PULP_TAG_KEY)
+        with pytest.raises(KeyError):
+            plugin_value_get(plugins, 'postbuild_plugins', PLUGIN_PULP_TAG_KEY, 'args')
+
+    @pytest.mark.parametrize('osbs', [OSBS_WITH_PULP_PARAMS], indirect=True)  # noqa:F811
+    def test_pulp_sync(self, osbs):
+        additional_params = {
+            'base_image': 'fedora:latest',
+        }
+        _, build_json = self.get_orchestrator_build_request(osbs, additional_params)
+        plugins = get_plugins_from_build_json(build_json)
+
+        args = plugin_value_get(plugins, 'postbuild_plugins', PLUGIN_PULP_SYNC_KEY, 'args')
+        expected_args = {
+            'publish': False,
+        }
+
+        assert args == expected_args
+
+    @pytest.mark.parametrize('osbs, use_pulp', [  # noqa:F811
+        ({'platform_descriptors': None, 'additional_config': None, 'kwargs': None}, False),
+        (OSBS_WITH_PULP_PARAMS, True)], indirect=['osbs'])
+    def test_pulp_publish(self, osbs, use_pulp):
```

Comment:
`use_pulp` seems to be unused here. If the outcome of the test is not expected to change based on that, is it necessary to parameterize it?

Reply:
Thanks, fixed.